python/ql/test/library-tests/regex/Regex.expected

```diff
@@ -207,9 +207,9 @@
 | ax{3,} | sequence | 0 | 6 |
 | ax{3} | char | 0 | 1 |
 | ax{3} | char | 1 | 2 |
-| ax{3} | char | 2 | 3 |
 | ax{3} | char | 3 | 4 |
 | ax{3} | char | 4 | 5 |
```

Comment:
I am not sure why 4-5 is in there, but both that and removing 2-3 is consistent with how `{n,m}` is handled..